plugins/gwxjg/gwxjg_data.ml

```diff
@@ -600,9 +600,10 @@ and unsafe_mk_semi_public_person conf base (p : Gwdb.person) =
   let surname = Tstr (E.surname base p) in
   let surname_aliases = mk_str_lst base (Gwdb.get_surnames_aliases p) in
   let events = Tlist [] in
+  let is_visible_by_visitors = Tbool (Util.is_public conf base p) in
```

Comment:
I think it should be `Util.is_fully_visible_to_visitors` we have on branch `private` instead.

we have two others `let access = Tbool (Util.is_public conf base p) in` in this file that should be `let access = Tbool (Util.is_fully_visible_to_visitors conf base p) in` too I think.